submodules/project_cleanup/variables.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "organization_id" {
+  description = "The organization ID whose projects to clean up"
+}
+
+variable "project_id" {
+  description = "The project ID to host the scheduled function in"
+}
+
+variable "region" {
+  description = "The region the project is in (App Engine specific)"
+  default     = "us-central1"
```

Comment:
This variable should not have a default value since it is dependent on the existence of an App Engine app which isn't managed by this module.